src/features/pages/home/index.tsx

```diff
@@ -11,13 +12,24 @@ import TrustpilotSection from './trustpilot'
 import { useOpenLiveChat } from 'components/hooks/use-open-live-chat-redirection'
 import MainRowNavigation from 'features/components/templates/navigation/main-nav'
 import MainFooter from 'features/components/templates/footer'
+
 const FastPaymentSection = lazy(() => import('./fast-payment'))
 const LiveMarketSection = lazy(() => import('./live-pricing-migration'))
 const TradeTypeSection = lazy(() => import('./trade-type'))
 const UserFriendlyPlatforms = lazy(() => import('./user-platfroms'))
 
 const HomePage = () => {
     useOpenLiveChat(true)
+    const third_party_config = Analytics?.getFeatureValue('third-party-config', {
+        is_livechat_enabled: true,
+        is_hotjar_enabled: true,
+        is_trustpilot_enabled: true,
+        is_snapchat_enabled: true,
+        is_tiktok_pixel_enabled: true,
+        is_gtm_enabled: true,
+        is_datadog_enabled: true,
```

Comment:
We do not need gtm, tiktok etc. 

Only social buttons visible in footer are configurable. Plus Whatsapp & LiveChat